core/admin/mailu/internal/nginx.py

```diff
@@ -26,8 +25,12 @@ def check_credentials(user, password, ip, protocol=None):
     if not user or not user.enabled or (protocol == "imap" and not user.enable_imap) or (protocol == "pop3" and not user.enable_pop):
         return False
     is_ok = False
```

Comment:
This was introduced by me... but not in that commit/PR. For the sake of keeping it simple, I suggest that we keep it as is and re-factor in another PR.

Do you have suggestions on what would be more meaningful? check_credentials(user, password, ...) returning is_ok doesn't sound that bad to me. The function checks both correctness (right password) AND validity (enabled for the specific protocol)... is_acceptable?

Reply:
> The function checks both correctness (right password) AND validity (enabled for the specific protocol)

The missing information is `password` here to avoid reader dig into `is_ok`. then what about `is_password_ok` ?

Reply:
password is probably not the right term either though... right now, it can be a password, a token or an SSO-token ... and if #783 goes through, it could be a lot of other things too

Reply:
then maybe `is_auth_ok` ? :)

Reply:
Ok, will send a separate PR once this is merged to refactor/rename